x-pack/plugins/security/server/plugin.ts

```diff
@@ -61,6 +51,8 @@ export interface SecurityPluginSetup {
   authz: Pick<Authorization, 'actions' | 'checkPrivilegesWithRequest' | 'mode'>;
   license: SecurityLicense;
 
+  audit: { getLogger: (id: string) => AuditLogger };
```

Comment:
Nit: what do you think about exposing the return type of the Audit Service setup and using that here? We do the same thing for Authentication and Authorization above.